src/eos-updater-fetch.c

```diff
@@ -208,7 +208,7 @@ content_fetch (GTask *task,
   goto cleanup;
 
  error:
-  g_message ("Fetch returning ERROR");
+  g_message ("Fetch: error: %s", error->message);
```

Comment:
I think it will be printed when `eos_updater_set_error` gets called by `content_fetch_finished` but I suppose it doesn't hurt to print it twice.

Reply:
Indeed. The alternative is to drop this message entirely. Printing ‘there was an error’ but not printing any details of the error is a bit rubbish. I’d be happy to drop this message if you think that’s a better approach?